docs/guides/connecting_to_your_data/overview.md

```diff
@@ -0,0 +1,205 @@
+- An InferredAssetDataConnector infers the `data_asset_name` by using a regex that takes advantage of patterns that exist in the filename or folder structure.  If your source data system is designed so that it can easily be parsed by regex, this will allow new data to be included by the Datasource automatically.
+- A ConfiguredAssetDataConnector, which allows you to have the most fine-tuning by requiring an explicit listing of each Data Asset you want to connect to.  This Data Connector would be ideal
+- A `RuntimeDataConnector` which enables you to use a `RuntimeBatchRequest` to wrap either an in-memory dataframe, filepath, or SQL query.
+
+In the `data_connectors` dictionary you may define multiple Data Connectors, including different types of Data Connectors, so long as they all have unique values in the place of the `<name_of_your_data_connector>` key.  We provide detailed guidance to help you decide on which Data Connectors to use in our guide: [How to choose which DataConnector to use](/docs/guides/connecting_to_your_data/how_to_choose_which_dataconnector_to_use).
+
+The `<additional_keys_based_on_source_data_system>` will be things like `base_directory` and `default_regex` for filesystems, or  `batch_identifiers` for SQL based data systems. For specifics on the additional keys that you can use in your Data Connectors' configurations, please see the corresponding guide for connecting to a specific source data system (since the keys you will need to define will depend on the source data system you are connecting to).
+
+### 3. Test your configuration
+
+Because the configurations for Datasources can vary depending on the underlying data system they are connecting to, Great Expectations provides a convenience function that will help you determine if there are errors in your configuration.  This function can be accessed from your Data Context, like so:
+
+<Tabs
+  groupId="yaml-or-python"
+  defaultValue='yaml'
+  values={[
+  {label: 'YAML', value:'yaml'},
+  {label: 'Python', value:'python'},
+  ]}>
+  <TabItem value="yaml">
+
+```python
+import great_expectations as ge
+
+datasource_yaml = ""  # Replace this with the yaml string you want to check for errors.
+
+context = ge.get_context()
+context.test_yaml_config(datasource_yaml)
+```
+
+</TabItem>
+<TabItem value="python">
+
+```python
+import great_expectations as ge
+from ruamel import yaml
+
+datasource_config = {}  # Replace this with the Python dictionary you want to check for errors.
+
+context = ge.get_context()
+context.test_yaml_config(yaml.dump(datasource_config))
+```
+
+</TabItem>
+</Tabs>
+
+### 4. Save the Datasource configuration to your Data Context.
+
+What is the point of configuring a Datasource if you can't easily use it in the future?  It this point you will want to save your Datasource configuration to your Data Context.  This can be done easily by using the `add_datasource()` function, which is conveniently accessible from your Data Context.  
```

Comment:
```suggestion
What is the point of configuring a Datasource if you can't easily use it in the future?  At this point you will want to save your Datasource configuration to your Data Context.  This can be done easily by using the `add_datasource()` function, which is conveniently accessible from your Data Context.  
```